documentation/Installation/PHP.md

```diff
@@ -147,7 +147,7 @@ Uncomment and fill in the keychain paths:
 ; OS-managed cert stores in its absence. If specified, this value may still
 ; be overridden on a per-stream basis via the "cafile" SSL stream context
 ; option.
-openssl.cafile = "/usr/local/etc/openssl@1.1/cert.pem"
+openssl.cafile = "/opt/homebrew/etc/openssl@1.1/cert.pem"
```

Comment:
This depends on the Mac processor:

* Intel: /usr/local
* Apple silicon: /opt/homebrew

Reply:
Is fixed in documentation by using the `$(brew --prefix)` function instead of fixed paths.